Libbna/CUSTOM-CMS
# #211: fixed login and register page on mobile view

**Testing Steps:-**

- Go to the login page and check the responsiveness of the page
- Do the same for registration page.
